src/plugins/ambient-mode/index.ts

```diff
@@ -53,7 +53,18 @@ export default createPlugin({
       const songImage = document.querySelector<HTMLImageElement>('#song-image');
       const songVideo = document.querySelector<HTMLDivElement>('#song-video');
       const image = songImage?.querySelector<HTMLImageElement>('yt-img-shadow > img');
-      const video = songVideo?.querySelector<HTMLVideoElement>('.html5-video-container > video');
+      const video = await waitForVideo();
+
+      async function waitForVideo() {
```

Comment:
see https://github.com/th-ch/youtube-music/blob/ebd167f3f2d39302b08fb9d3477ed173890a3f61/src/plugins/album-actions/index.ts#L186

Reply:
Hello,

I tried to comply with your request, however I am not able to edit the code to your wishes, due to my lack of knowledge and experience. My current version involved a lot of experimenting as well as asking ChatGPT for help. 

So if anyone else would be able to take a look and edit it accordingly, that would be great. If not, please close this request and I will continue to build the app myself with my suboptimal fix.